### DISCUSSION TITLE
这个项目稳定吗？有文档嘛？

### DISCUSSION THREAD

**Discussion by Zhou-Shilin** [LagrangeDev/Lagrange.Core#20] [Q&A] [answered]
如题，看到go-cqhttp的迁移建议，不知道这个好不好用…

**Comment by IMZCC** [comment]
Action Build下了二进制程序，但是不知道怎么用

**Comment by ghost** [comment]
可以加群下载那个appsettings.json

**Reply by ChrisKimZHT** [reply]
所以群号是什么呢？仓库里文档是一点没有，README 也是英文的，是故意提高门槛嘛。

**Reply by Zhou-Shilin** [reply]
群号README里面有的，348981074
全英文的倒也不是完全看不懂，但是好歹有个文档吧？问了群里的开发者，说是自己看源代码解决💦
我是直接换到隔壁chronocat了

@ChrisKimZHT

**Comment by Zhou-Shilin** [comment, marked as answer]
自问自答：这个项目并不完善，但是能用；没有文档，需要看源代码自己解决
个人直接换chronocat了，等项目完善了可能再回来吧

**Reply by eric2788** [reply]
chronocat现在已经寄了，你是最后的希望

**Reply by Zhou-Shilin** [reply]
现在在试openshamrock
